include/nbl/video/IPhysicalDevice.h

```diff
@@ -522,7 +523,7 @@ class NBL_API2 IPhysicalDevice : public core::Interface, public core::Unmovable
         struct FormatPromotionRequest
         {
             asset::E_FORMAT originalFormat = asset::EF_UNKNOWN;
-            FORMAT_USAGE usages;
+            FORMAT_USAGE usages = FORMAT_USAGE(0);
```

Comment:
also why isn't this field a `core::bitflag` ?

Reply:
It's the Usage struct with the 1 bit members and stuff. see `IPhysicalDevice::SFormatImageUsage`

Reply:
🤦 missed the `template<typename FORMAT_USAGE>` above